server.py

```diff
@@ -105,17 +107,9 @@ def compare_talks(talk1_id, talk2_id):
     duration2.append(talk2.duration_min)
     duration2.append(talk2.duration_sec)
     if talk1 and talk2:
+        # Reason for redundancy?
         return jsonify({"status": "success",
-                        "talk_name1": talk1.talk_name,
-                        "num_comments1": (f"{talk1.num_comments:,d}"),
-                        "num_views1": (f"{talk1.num_views:,d}"),
-                        "duration1": duration1,
-                        "talk_name2": talk2.talk_name,
-                        "num_comments2": (f"{talk2.num_comments:,d}"),
-                        "num_views2": (f"{talk2.num_views:,d}"),
-                        "duration2": duration2,
-                        "rating_list" : rating_list
-        })
+                        "talks": [talk1, talk2]})
     else:
         return jsonify({"status": "error",

```

Comment:
I love this idea and I've seen you use it in other routes. But if its not consistently the same response object, it leads to bugs and more unnecessary code. 

The pattern can translate into: `jsonify(status='success', message='Some message: data=[])` See https://flask.palletsprojects.com/en/1.1.x/api/#flask.json.jsonify

With JSON responses its good practice to return the same object with the same properties to keep the API uniform and standardized.